examples/SESAR/test/iSamplesEOI00002HBasic.json

```diff
@@ -3,20 +3,20 @@
     "@id": "https://data.isamples.org/digitalsample/igsn/EOI00002H",
     "label": "J730-GTHFS-16",
     "sampleidentifier": "igsn:EOI00002H",
-    "description": "",
+    "description": "Not Provided",
     "hasContextCategory": ["Subsurface fluid reservoir"],
     "hasMaterialCategory": ["Gaseous material"],
     "hasSpecimenCategory": ["Container with fluid"],
     "keywords": ["Individual Sample"],
     "producedBy": {
         "label": "Sampler:Fluid:GTHFS",
-        "description": "cruiseFieldPrgrm:TN300. launchPlatformName:Jason II. Sampler:Fluid:GTHFS.  HFS gastight. Red-center-9. T=250C. launch type:ROV, navigation type:USBL",
+        "description": "cruiseFieldPrgrm:TN300. launchPlatformName:Jason II. Sampler:Fluid:GTHFS. HFS gastight. Red-center-9. T=250C. launch type:ROV, navigation type:USBL",
         "hasFeatureOfInterest": "volcano",
-        "responsibility": ["Evans Leigh,,Collector","Andra Bobbitt,,Sample Owner"],
+        "responsibility": ["Evans_Leigh,,Collector","Andra Bobbitt,,Sample Owner"],
```

Comment:
I don't think we should be in the business of deciding when to remove underscores, should we?  If we are, where does the line end?  What do we consider in need of transformation and what do we pass through.  For now, I just changed the sample because that was simpler.